golem-worker-service/src/api/worker.rs

```diff
@@ -47,7 +30,7 @@ impl WorkerApi {
         &self,
         template_id: Path<TemplateId>,
         request: Json<WorkerCreationRequest>,
-    ) -> Result<Json<VersionedWorkerId>> {
+    ) -> Result<Json<WorkerId>> {
```

Comment:
similar like i mentioned in case of gRpc api 
instead of just `WorkerId`, have some payload 
```rust
struct WorkerCreationResponse {
    worker_id: WorkerId,
    template_version: u64
}

```